shuup/core/models/_suppliers.py

```diff
@@ -83,9 +88,10 @@ class Supplier(ModuleInterface, TranslatableShuupModel):
     translations = TranslatedFields(
         description=models.TextField(blank=True, verbose_name=_("description"))
     )
+    deleted = models.BooleanField(default=False, verbose_name=_("deleted"))
 
     search_fields = ["name"]
-    objects = SupplierQueryset.as_manager()
+    objects = SupplierManager()
```

Comment:
Really doesn't sound like changing this benefits us too much. If `filter(deleted=False)` is now added every single queryset how one get the deleted items? I think you could filter out deleted directly in the admin views and then also here in enabled queryset-manager.

Reply:
ok, will revert SupplierQueryset